ci.sh

```diff
@@ -2,8 +2,6 @@
 
 set -exuo pipefail
 
-sudo yum install -y podman make golang rsync
-
```

Comment:
This is tracked in https://github.com/ironcladlou/openshift4-libvirt-gcp/pull/45 which isn't merged yet, probably better to keep this commit out of this PR?

Reply:
@cfergeau this is not merged but the image is already updated to GCP with those changes, I don't see any strong opinion to not merge that PR.